docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md

```diff
@@ -0,0 +1,153 @@
+
+- **Default** - Toggle to select as the default name in dashboard panels. When you go to a dashboard panel this will be the default selected data source.
+
+### HTTP section
+
+- **URL** - The URL of your Elasticsearch server. If your Elasticsearch server is local, use `<http://localhost:9200>`. If it is on a server within a network, this is the URL with port where you are running Elasticsearch. Example: `<http://elasticsearch.example.orgname:9200>`.
+
+- **Allowed cookies** - Specify cookies by name that should be forwarded to the data source. The Grafana proxy deletes all forwarded cookies by default.
+
+- **Timeout** - The HTTP request timeout. This must be in seconds. There is no default, so this setting is up to you.
+
+### Auth section
+
+There are several authentication methods you can choose in the Authentication section.
+
+{{% admonition type="note" %}}
+Use TLS (Transport Layer Security) for an additional layer of security when working with Elasticsearch. For information on setting up TLS encryption with Elasticsearch see [Configure TLS](https://www.elastic.co/guide/en/elasticsearch/reference/8.8/configuring-tls.html#configuring-tls). You must add TLS settings to your Elasticsearch configuration file **prior** to setting these options in Grafana.
+{{% /admonition %}}
+
+- **Basic authentication** - The most common authentication method. Use your `data source` user name and `data source` password to connect.
+
+- **With credentials** - Toggle to enable credentials such as cookies or auth headers to be sent with cross-site requests.
+
+- **TLS client authentication** - Toggle to use client authentication. When enabled, add the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server.
+
+- **With CA cert** - Toggle to authenticate with a CA certificate. Follow the instructions of the CA (Certificate Authority) to download the certificate file.
+
+- **Skip TLS verify** - Toggle on to bypass TLS certificate validation.
+
+- **Forward OAuth identity** - Forward the OAuth access token (and also the OIDC ID token if available) of the user querying the data source.
+
+### Custom HTTP headers
+
+- **Header** - Add a custom header. This allows custom headers to be passed based on the needs of your Elasticsearch instance.
+
+- **Value** - The value of the header.
+
+### Elasticsearch details
+
+The following settings are specific to the Elasticsearch data source.
+
+- **Index name** - Use the index settings to specify a default for the `time field` and your Elasticsearch index's name. You can use a time pattern, such as `YYYY.MM.DD`, or a wildcard for the index name.
+
+- **Pattern** - Select the matching pattern if using one in your index name. Options include:
```

Comment:
Are these options visible in the UI? If so, I would remove the bulleted list. Documenting lists of values can become a maintenance issue (if, for instance, a 15 min interval is added to the UI).